components/invitation_modal/invite_view.scss

```diff
@@ -10,13 +17,53 @@
     }
 
     &__copyLink {
+        &.tertiary-button {
```

Comment:
great button design implementation. I think this tertiary button is going to be widely used with the new designs, I wonder if we could put this in a mixin into `_mixins.scss` so it can be reused?

Reply:
Thanks to the [playbooks](https://github.com/mattermost/mattermost-plugin-playbooks/blob/ef3ae7be3baeca825a8f16017cd8715e6b64cf86/webapp/src/components/assets/buttons.tsx#L77) people! Will do. I think the long term hope is to consume this through a compass component that encapsulates it, but having it as a mixin is better in the interim.

Reply:
thanks!